src/ray/common/ray_config_def.h

```diff
@@ -60,6 +60,11 @@ RAY_CONFIG(uint64_t, num_heartbeats_warning, 5)
 
 /// The duration between reporting resources sent by the raylets.
 RAY_CONFIG(uint64_t, raylet_report_resources_period_milliseconds, 100)
+RAY_CONFIG(uint64_t, raylet_report_loads_period_milliseconds, 1000)
```

Comment:
I search the codes and found it seems not used now. (correct me if i'm wrong)
If it is used, change the default value from 100 to 1000 will make differences in resource syncing.

Reply:
This is only used for autoscaler for now I think. Given that 1s actually is nothing compared with new nodes up, it's OK. Otherwise, the resource should be put in the resource manager.
A better way is to introduce versioning for the loads resources.